### PR TITLE
[IRC] Rework interacting with irc server

### PR DESCRIPTION
**Pull request by avano**
//test: `@irc`

### CODE REVIEW

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/IrcValidationSteps.java [hunk 1]
```diff
@@ -2,131 +2,52 @@
...
-        connection = new IRCConnection(StringUtils.substringBetween(TestConfiguration.openShiftUrl(), "//", ":"), new int[] {31111},
-                null, "checker", "checker", null);
-        listener = new IrcListener();
-        connection.addIRCEventListener(listener);
-        connection.connect();
-        Thread.sleep(15000L);
-        for (String chan : channels.split(",")) {
-            connection.doJoin(chan);
-        }
+    private static final String CONTROLLER_ROUTE = "http://" + OpenShiftUtils.getInstance().routes()
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/IrcValidationSteps.java]: How are we sure that the route exists before the class is loaded?

Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/bdd/validation/IrcValidationSteps.java]: Only this way, that it is basically the first step of the tests:

https://github.com/syndesisio/syndesis-qe/blob/c442b4ae8c9a32f6317c0a38ef1eb7544c070120/rest-tests/src/test/resources/features/irc.feature#L7-L9

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**